### PR TITLE
Cse 347-update-style-on-target-page-title-bar

### PR DESCRIPTION
**Pull request by Bensonre**
# Changes introduced
added font-awesome icons to npm packages, I think this is where the merge conflict is originating
change download icon to download cloud
fixed the positioning of title bar information

## Related issue(s):

- CSE-_user story or bug_
- CSE-_task number_

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [ ] Added new stories for created/modified components (if applicable)
- [ ] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

File: src/components/TitleBar/DownloadBtn.tsx [hunk 1]
```diff
@@ -50,20 +52,32 @@ export class DownloadBtn extends Component<DownloadBtnProps, DownloadBtnState> {
...
+            font-size: 15px;
+          }
+          #download-btn-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            flex-direction: column;
+            color: White;
+          }
+          a:hover {
```
Comment (danvanhorn) [inline src/components/TitleBar/DownloadBtn.tsx]: We need to remove this hover tag. We're not doing any hover selectors in the app

**Review by danvanhorn — CHANGES_REQUESTED** [review]
We only need to address my previous comment.

File: src/components/TitleBar/DownloadBtn.tsx (hunk 1 above)
Comment (Bensonre) [reply src/components/TitleBar/DownloadBtn.tsx]: done

**Review by danvanhorn — APPROVED** [review]
Everything is in order, good job.

**Pull request merged**